docker-compose.yml

```diff
@@ -1,33 +1,36 @@
+version: '3.0'
+
 services:
   db:
+    image: timescale/timescaledb:latest-pg12
```

Comment:
Can you make the db use `timescaledev/promscale-extension:latest-pg12`? It's the image with the promscale extension included for better performance?

Reply:
Oh I had no idea this image exists haha, I thought I had to have 2 containers :sweat_smile: 

That could help people !

Reply:
You still need the two containers. 
timescale/promscale is the prometheus connector 
timescaledev/promscale-extension is postgres+timescaledb+"promscale-extension" 

This extension is a postgres extension that contains code that helps promscale do ingest and querying multiple times faster.

Reply:
you should revert to using the two containers, just change the image for the db

Reply:
Hum, I feel like this could benefit from having a bit more of documentation. I'll update it and push it